source/deployment/sunstone_setup/suns_advance.rst

```diff
@@ -119,13 +119,15 @@ We will provide the instructions for Apache web server but the steps will be sim
 
 First thing you have to do is install Phusion Passenger. For this you can use pre-made packages for your distribution or follow the `installation instructions <https://www.phusionpassenger.com/download/#open_source>`__ from their web page. The installation is self explanatory and will guide you in all the process, follow them an you will be ready to run Sunstone.
 
-Next thing we have to do is configure the virtual host that will run our Sunstone server. We have to point to the ``public`` directory from the Sunstone installation, here is an example:
+Next thing we have to do is configure the virtual host that will run our Sunstone server. We have to point to the ``public`` directory from the Sunstone installation, here is an example of /etc/apache2/sites-available/opennebula.conf:
+
+
 
 .. code::
 
     <VirtualHost *:80>
       ServerName sunstone-server
-      PassengerUser oneadmin
+      PassengerUser <username>
```

Comment:
I see where this is coming from. It is important to focus the guide for admins/user more than developer.